src/main/java/Entities/Player.java

```diff
@@ -235,9 +305,20 @@ else if(a == b && (consecutive + 1) == 3){
      * @param property the property to build the house on
      * @param houses the number of houses to build
      */
-    public void buildHouse(Property property, int houses) {
-        if (properties.contains(property)) {
-            property.addHouse(this, houses);
+    public String buildHouse(Property property, int houses) {
+        switch (property.addHouse(this, houses)) {
```

Comment:
Good edition to check the type of the property before adding houses and good choice to use the switch keyword.